deps/Makefile

```diff
@@ -441,6 +446,9 @@ endif # ARCH == ppc64
 
 
 ifeq ($(LLVM_SANITIZE),1)
+# NOTE: LLVM_SANITIZE + USE_LLVM_SHLIB only works with LLVM_USE_CMAKE, because
```

Comment:
Maybe turn this into an actual warning/error when incompatible options are passed?